mocha-client-tests/addon/tests/lib/background-messaging.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function(done) {
+            chrome.runtime.sendMessage({action: 'ping'},function(response) {
+                expect(response).to.equal('pong');
```

Comment:
https://github.com/mochajs/mocha/issues/1128 - looks like I'm not sure why it wait until fail